pkg/cache/manager.go

```diff
@@ -37,6 +37,7 @@ type Manager struct {
 }
 
 type Opt struct {
+	Disabled bool
```

Comment:
Good catch! Maybe also update these `config.toml`:

- misc/snapshotter/config.toml
- tests/e2e/k8s/snapshotter-cri.yaml
- tests/e2e/k8s/snapshotter-kubeconf.yaml

Reply:
`cache_manager.disable` is still set to be `false` for those three files. I'm keeping the configuration item for compatibility though it doesn't take effect. Not sure whether `cache_manager.disable` will be used to control `auto garbage collect` in future or not.

Reply:
After a second thought, I think we can support `recyclebin` to keep deleted files for a while. Those configuration item can be used for it.

Reply:
Introducing recyclebin seems to add complexity and doesn't feel necessary, it is better to let kubelet & containerd control the GC policy.